src/streaming/MediaPlayer.js

```diff
@@ -124,6 +125,7 @@ function MediaPlayer() {
         mediaPlayerInitialized = false;
         playbackInitialized = false;
         autoPlay = true;
+        skipDataLoss = false;
```

Comment:
I think makes more sense declaring this in MediaPlayerModel as we are doing for other configuration parameters.

Reply:
Could we change this name to be skipGaps?